server/controller/taskController.js

```diff
@@ -0,0 +1,54 @@
+const Task = require('../db/models/Task');
+const User = require('../db/models/User');
```

Comment:
why do you need models here ?
I think this can be deleted

Reply:
Deleted